.github/CONTRIBUTING.md

```diff
@@ -7,3 +7,8 @@ PsySH follows [PSR-1](http://php-fig.org/psr/psr-1/) and [PSR-2](http://php-fig.
 ## Branching model
 
 Please branch off and send pull requests to the `develop` branch.
+
+## Running tests
+
+1. Run `composer bin phpunit install` to install PHPUnit.
```

Comment:
how about moving this into the `Makefile`?

```
PHPUNIT=vendor/bin/phpunit

.PHONY test
test: $(PHPUNIT)

$(PHPUNIT)t: vendor-bin/phpunit
	composer bin phpunit install
	touch $@
```

This way the DX is not affected by it either

Reply:
Yeah, makes sense.